src/components/common/CustomDialog.js

```diff
@@ -22,6 +32,11 @@ const CustomDialog = ({ children, visible, title, message, dismissText, onDismis
         {message ? <Paragraph>{message}</Paragraph> : null}
       </Dialog.Content>
       <Dialog.Actions>
+        {onCancel ? (
```

Comment:
Instead of question mark `?`, I would prefer `&&`...so it's like:
```jsx
{onCancel && (
    <CustomButton onPress={onCancel} disabled={loading} loading={loading}>
        Cancel
    </CustomButton>
)}
```

Reply:
no, I usually use && when there is a value for `if` but not for `else`...so in this case `else` return null...anyway it's only about stylecode, and not about functionality

Reply:
I ment i've changed all "?" to "&&"
check the next commit

Reply:
👍